src/data/encoding-test.lisp

```diff
@@ -232,9 +232,9 @@
 ;; for some reason (make-hash-table) is not equalp
 ;; to (make-hash-table) with ecl.
 
-#-openmcl(test-encoding hash.1 (make-hash-table))
+#-openmcl(test-encoding hash.1 (make-hash-table :size 0))
```

Comment:
cl-store which is used for tests also store the hash-table's size slot. Ideally this shouldn't be used to check congruence, since the size is more an internal performance detail. But this fixes the tests on LW for now.